src/components/TransactionItem/TransactionIncome/styled.tsx

```diff
@@ -22,6 +22,9 @@ export const TransactionCapacityValuePanel = styled.div<{ increased: boolean }>`
   justify-content: flex-end;
   color: ${props => (props.increased ? props.theme.primary : '#FF6347')};
   font-size: 16px;
+  .monospace {
+    color: inherit;
+  }
```

Comment:
This commit has not been removed.

Reply:
Redundant style was removed by https://github.com/Magickbase/ckb-explorer-frontend/pull/181/commits/85fbdad61888c938eabea380d8960be06e1393db